app/views/hq/sponsors/_index.html.haml

```diff
@@ -0,0 +1,43 @@
+- if sponsors.empty?
+  .table-responsive
+    %table.table
+      %tbody
+        %tr
+          %td
+            No Sponsors found
```

Comment:
Given there is no tabulated data to show, something like a panel would be much more appropriate.
http://getbootstrap.com/components/#panels